avalanche/models/dynamic_modules.py

```diff
@@ -30,7 +29,7 @@ class DynamicModule(Module):
     `model_adaptation`, which adapts the model given the current experience.
     """
 
-    def adaptation(self, experience: CLExperience):
+    def adaptation(self, experience: CLExperience = None):
```

Comment:
Why do you need this to be None?

Reply:
that's definitely a typo, I never intended to modify that class.